pulsar-client-go/pulsar/message.go

```diff
@@ -42,6 +42,9 @@ type ProducerMessage struct {
 
 	// Set the sequence id to assign to the current message
 	SequenceID int64
+
+	// Set the time to deliver the message
+	DeliverAtTime time.Time
```

Comment:
In other clients we've also added the `DeliverAfter` option to specify a delay.

Reply:
Sorry- somehow I missed all notifications on these comments. I will address them shortly. Thanks!